src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp

```diff
@@ -1665,11 +1698,14 @@ SubCmdValidate::execute(const SubCmdOptions& _options) const
     }
 
     // Must be a test name, look to see if should be added
-    const std::string testSuiteName = boost::algorithm::to_lower_copy(testSuite[index].ptTest.get("name",""));
+    const std::string fullTestSuiteName = testSuite[index].ptTest.get("name","<unknown>");
+    std::vector<std::string> testSuiteNames = get_run_test_names(fullTestSuiteName);
     for (const auto & testName : testsToRun) {
-      if (testName.compare(testSuiteName) == 0) {
-        testObjectsToRun.push_back(&testSuite[index]);
-        break;
+      for (const auto & testSuiteName : testSuiteNames){
+        if (testName.compare(boost::algorithm::to_lower_copy(testSuiteName)) == 0) {
```

Comment:
```suggestion
        if (boost::iequals(testname, testSuiteName)) {
```

Reply:
I thought we did this to support how case in windows doesn't matter. Otherwise I think this is much nicer.

Reply:
Didn't think about windows, was simply noticing that this case was "attempting" to do a case insensitive compare.